### DISCUSSION TITLE
Can I limit devs apps list in builder ?

### DISCUSSION THREAD

**Discussion by mielune** [Budibase/budibase#5361] [Help] [answered]
Hi community,

I installed a selft-hosted BB at home, for me and kids.
In the builder, all the dev users have access to all the apps, and are allowed to edit thems. I would like to hide my personnal projects from kids in the builder; I don't want to a kids to break my home automation apps ;) 
Can i do this ?

**Comment by melohagan** [comment, marked as answer]
Hey @mielune 

Assuming you're the admin, then you should be able to configure the roles of your kid accounts. 
https://docs.budibase.com/docs/user-management

So you would select **No Access** for each app that you don't want the kids to see.

![Screenshot 2022-04-12 at 13 01 25](https://user-images.githubusercontent.com/101575380/162958193-a70e7096-c155-4536-bd63-23a64fb665d1.png)

There does appear to be a bug however in that you cannot set a user role to No Access for dev users, which I have raised here: https://github.com/Budibase/budibase/issues/5367

**Comment by mielune** [comment]
Thank you @melohagan  

As a nocode platform, my kids are Dev, I try to push them in the dev world ^^
I will track this issue ;)

Thank you

**Comment by mielune** [comment]
Do you planned to a groups management / platforme roles management in your road map ? (for example, not to allow all Dev to be Admin "by default" for all apps when they are added to BB).
I have 2 kids so it is easy for me, but for a company ^^ Lot of kids :D

**Reply by melohagan** [reply]
There's an issue here relating to setting default roles: https://github.com/Budibase/budibase/issues/4160

And also in the roadmap itself: https://github.com/Budibase/budibase/projects/10
There is a further role enhancements epic in the Q3 column
